homework 4.5

timmywilson opened the pull request.

timmywilson committed:
homework 4

timmywilson committed:
next set of changes

eikonomega commented:
Hey @timmywilson I think you've got something funky going on with your line-endings.  Probably a PC issue.  We can look at it together in class.

timmywilson committed:
Lost...

Merge remote-tracking branch 'upstream/master'

Conflicts:
	dragon-scrolls/level-2/exercise-5.md
	dragon-scrolls/level-2/exercise-7.md
	dragon-scrolls/level-2/readme.md
	dragon-scrolls/level-2/teaching-notes.md
	dragon-scrolls/level-3/exercise-04.md
	dragon-scrolls/level-3/exercise-1.md
	dragon-scrolls/level-3/readme.md
	training/level-2-command-line-interfaces/bfp-reference/cli_basics.py
	training/level-2-command-line-interfaces/bfp-reference/exercise_07/file_ops.py
	training/level-2-command-line-interfaces/bfp-reference/exercise_07/stdlib_cli.py
	training/level-2-command-line-interfaces/bfp-reference/multipurpose_cli.py
	training/level-2-command-line-interfaces/dragon-warrior/ndrk_stdlib_cli.py

timmywilson committed:
Saving files before refreshing line endings

timmywilson committed:
Normalize all the line endings
